services/src/main/java/org/keycloak/broker/saml/mappers/AttributeToRoleMapper.java

```diff
@@ -56,13 +57,13 @@
         property = new ProviderConfigProperty();
         property.setName(ATTRIBUTE_NAME);
         property.setLabel("Attribute Name");
-        property.setHelpText("Name of attribute to search for in assertion.  You can leave this blank and specify a friendly name instead.");
+        property.setHelpText("Name-attribute to search for in assertion Attribute elements. You can leave this blank and specify a friendly name instead.");
         property.setType(ProviderConfigProperty.STRING_TYPE);
         configProperties.add(property);
         property = new ProviderConfigProperty();
         property.setName(ATTRIBUTE_FRIENDLY_NAME);
         property.setLabel("Friendly Name");
-        property.setHelpText("Friendly name of attribute to search for in assertion.  You can leave this blank and specify a name instead.");
+        property.setHelpText("FriendlyName-attribute to search for in assertion Attribute elements. You can leave this blank and specify a name instead.");
```

Comment:
The help text should be improved, name vs friendly name is not as clear as it could be.